specification/sbi_atee.adoc

```diff
@@ -556,14 +556,33 @@ The possible error codes returned in `sbiret.error` are shown below.
 ------
 struct sbiret sbi_tee_host_finalize_tvm(unsigned long tvm_guest_id,
                                         unsigned long entry_sepc,
-                                        unsigned long entry_arg);
+                                        unsigned long entry_arg,
+                                        unsigned long tvm_identity_addr);
 ------
 Transitions the TVM specified by `tvm_guest_id` from the `TVM_INITIALIZING` state to a `TVM_RUNNABLE`
 state. Also, sets the entry point (`ENTRY_PC`) using `entry_sepc` and boot argument (`ENTRY_ARG`) 
 using `entry_arg` for the boot VCPU. Both `entry_sepc` and `entry_arg` are included in the measurement
-of the TVM. 'entry_sepc' is the address in TVM binary to start the boot VCPU from and `entry_arg` is 
+of the TVM. `entry_sepc` is the address in TVM binary to start the boot VCPU from and `entry_arg` is
 the address of guest fdt and is passed as an argument to the boot VCPU in `a1` GPR. 
 
+`tvm_identity_addr` points to a 64 bytes buffer containing a host-defined TVM
+identity. This piece of data can be used to bind TVMs to a host-defined identity
+(e.g. an attestation service public key, a guest configuration file hash, an
+attestation policy description, etc). Although this piece of data is included in
+the TVM attestation certificate as a dedicated TVM claim (`tvm-identity`), it is
+*not* included in the TVM measurements.
+That allows for the host to personalize cryptographically identical TVMs through
+an attestable and verifiable identity.
+
+The semantics of this piece of data is defined by the host and can be ignored
+by both the guest and the attestation services. However, when being used, the
+TVM identity can be leveraged as follows:
```

Comment:
Is the `tvm_identity` optional or required?

Reply:
I think this needs to be optional

Reply:
I think this needs to be optional

Reply:
Yes, it should be optional and I updated the PR accordingly.